tic-tac-toe/src/index.css

```diff
@@ -0,0 +1,94 @@
+body {
+    font: 14px "Century Gothic", Futura, sans-serif;
+    margin: 40px;
+    background: #fff;
+}
+
+ol, ul {
+    padding-left: 30px;
+}
+
+.board-row:after {
+    clear: both;
+    content: "";
+    display: table;
+}
+
+.status {
```

Comment:
Looks like this status style isn't used in the app. If that's the case, then this can be removed.